src/include/daos_srv/vos.h

```diff
@@ -81,6 +123,68 @@ vos_dtx_list_cos(daos_handle_t coh, daos_unit_oid_t *oid,
 int
 vos_dtx_list_committable(daos_handle_t coh, struct daos_tx_entry **dtes);
 
+/**
+ * Check whether the specified DTX can be committed or not.
+ *
+ * \param coh	[IN]	Container open handle.
+ * \param dti	[IN]	The DTX identifier.
+ *
+ * \return		DTX_ST_INIT	Related DTX does not exists, or it is
+ *					initialized, but the modification has
+ *					not completed yet.
+ *			DTX_ST_PREPARED	means the DTX has been 'prepared', so
+ *					the modification has been done locally
+ *					on related replica(s).
```

Comment:
so to go back to my original comment, does PREPARED mean it "can be committed" whereas INIT and COMMITTED mean it can't (or already has been so doesn't need to be) ?

Reply:
'prepared' means that the modification for current replica is ready, but only when all the replicas all be 'prepared', then the whole transaction can be committed. So as long as one replica claims as 'committed', then it means that the transaction has even been committed and such replica has made the commit.